types/hapi/tsconfig.json

```diff
@@ -6,7 +6,7 @@
         ],
         "noImplicitAny": true,
         "noImplicitThis": true,
-        "strictNullChecks": true,
+        "strictNullChecks": false,
```

Comment:
why?

Reply:
We should absolutely keep strictNullChecks set to true

Reply:
For my app really need this to stay set to true

Reply:
Yea, revert this.

Reply:
Commit 8d1ffa7

Reply:
@rafaelsouzaf you changed `strictFunctionTypes` rather than `strictNullChecks`

Reply:
Oh my... sorry! Fixed. Thanks @corbinu now I understand your avatar :)